Description: install libs into ${INSTALL_LIB_DIR}, needed for multiarch
Author: Anton Gladky <gladk@debian.org>
Last-Update: 2013-03-15

--- a/wrappers/java/CMakeLists.txt
+++ b/wrappers/java/CMakeLists.txt
@@ -73,6 +73,6 @@
   COMMAND ${CMAKE_COMMAND} -E copy 
     ${CMAKE_CURRENT_SOURCE_DIR}/WrappingJava/build.xml 
     ${CMAKE_CURRENT_BINARY_DIR}/WrappingJava/)
-install(TARGETS WrapGmsh DESTINATION lib OPTIONAL)
+install(TARGETS WrapGmsh DESTINATION ${INSTALL_LIB_DIR} OPTIONAL)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/WrappingJava/WrapGmsh.jar 
-        DESTINATION lib)
+  DESTINATION ${INSTALL_LIB_DIR})
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1334,7 +1334,7 @@
 # (e.g. "make lib" or "make shared" followed by "make install/fast")
 install(TARGETS gmsh DESTINATION ${GMSH_BIN} OPTIONAL)
 if(ENABLE_BUILD_LIB)
-  install(TARGETS lib DESTINATION ${GMSH_LIB} OPTIONAL)
+  install(TARGETS lib DESTINATION ${INSTALL_LIB_DIR} OPTIONAL)
 endif(ENABLE_BUILD_LIB)
 if(ENABLE_BUILD_SHARED OR ENABLE_BUILD_DYNAMIC)
   if(WIN32 AND NOT MSVC OR CYGWIN)
@@ -1353,7 +1353,7 @@
       install(FILES ${GCC_DLL} ${GFORTRAN_DLL} ${STDC_DLL} DESTINATION ${GMSH_LIB})
     endif(GCC_DLL AND GFORTRAN_DLL AND STDC_DLL)
   else(WIN32 AND NOT MSVC OR CYGWIN)
-    install(TARGETS shared DESTINATION ${GMSH_LIB} OPTIONAL)
+    install(TARGETS shared DESTINATION ${INSTALL_LIB_DIR} OPTIONAL)
   endif(WIN32 AND NOT MSVC OR CYGWIN)
 endif(ENABLE_BUILD_SHARED OR ENABLE_BUILD_DYNAMIC)
 
